engine/src/flutter/lib/ui/window/platform_configuration.cc

```diff
@@ -77,6 +77,13 @@ void PlatformConfiguration::DidCreateIsolate() {
   dispatch_pointer_data_packet_.Set(
       tonic::DartState::Current(),
       Dart_GetField(library, tonic::ToDart("_dispatchPointerDataPacket")));
+  // The embedded platform view (e.g. UIView on iOS) is called "platform view"
+  // on framework side, but "embedded view" on engine side. On the other hand,
+  // "platform view" refers to the whole flutter view on engine side.
+  embedded_view_should_accept_gesture_.Set(
```

Comment:
Nit: Rename to "embedded_native_view..." to keep inline
```suggestion
  embedded_native_view_should_accept_gesture_.Set(
```